src/components/canvas/Card/Card.tsx

```diff
@@ -76,7 +82,4 @@ class Card extends Component<CardProps> {
   }
 }
 
-Card.Header = CardHeader;
-Card.Body = CardBody;
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** Why are these being removed? Won't it break all instances of `<Card.Body></Card.Body>`‏